digitalpy/core/zmanager/impl/zmq_pusher.py

```diff
@@ -0,0 +1,44 @@
+import zmq
+from digitalpy.core.zmanager.pusher import Pusher
+from digitalpy.core.zmanager.request import Request
+from digitalpy.core.parsing.formatter import Formatter
+
+class ZMQPusher(Pusher):
+    
+    def __init__(self, formatter: Formatter):
+        self.pusher_context = None
+        self.pusher_socket = None
+        self.pusher_formatter = formatter
+
+    def initiate_connections(self, port: int, address: str):
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D102 Missing docstring in public method